pdcupdater/utils.py

```diff
@@ -694,3 +696,23 @@ def inner(*args, **kwargs):
                     time.sleep(interval)
         return inner
     return wrapper
+
+
+def get_token(pdc_api_url, keytab):
+    """
+    Uses kerberos keytab for automatic authentication 
+    and retrieves the token from pdc-updater
+    """
+    if not os.path.exists(keytab):
+        raise OSError("Keytab file not found")
+
+    os.environ['KRB5_CLIENT_KTNAME'] = 'FILE:{0}'.format(keytab)
```

Comment:
@sochotnicky he actually tested it using the dev keytab for pdc-updater. It seems to work.